# Finding the Sum of Cubed Numbers

I was reading an article in ScienceAlert (link: https://www.sciencealert.com/mathematicians-solve-a-long-standing-42-problem-using-planetary-supercomputer) and was inspired by the revelation of the number 42 having three numbers being cubed. The surely were not straightforward numbers, but the whole thought behind it was kind of inspiring. 

In [18]:
x = int(input("What is the x-value you are inputting?"))
y = int(input("What is the y-value you are inputting?"))
z = int(input("What is the z-value you are inputting?"))
k = (x**3)+(y**3)+(z**3)
print("The result of the cubic sum is:",k)

What is the x-value you are inputting?-80538738812075974
What is the y-value you are inputting?+80435758145817515
What is the z-value you are inputting?+12602123297335631
The result of the cubic sum is: 42


***They found magic in these 3 numbers:***

-80538738812075974, +80435758145817515 and +12602123297335631 

***So the full equation can be viewed visually as follows:***

k = (-80538738812075974)^3 + (80435758145817515)^3 + (12602123297335631)^3 = 42

In case of the people who are typically uninterested in Mathematics, the key behind this whole inspiration for Mathematicians around the world was that in 1954, they wanted to determine the x,y,z for every natural number such that, when you cube x,y,z you can get a natural number. Completing 42 was significant because, it now completes the numbers from 1 to 100. The focus now goes up by a notch: 1 to 1000. Right now we still need the cubic sums of 114, 165, 390, 579, 627, 633, 732, 906, 921 and 975.

If anyone has a supercomputer, hit me up! :)